third_party/amd/backend/compiler.py

```diff
@@ -396,7 +395,14 @@ def make_amdgcn(src, metadata, options):
         assert len(names) == 1
         metadata["name"] = names[0]
         # llvm -> hsaco
-        amdgcn = llvm.translate_to_asm(src, amd.TARGET_TRIPLE, options.arch, '', [], options.enable_fp_fusion, False)
+        flags = []
+        # XXX(Kyle): The sink-insts-to-avoid-spills helps mitigate register spills
```

Comment:
Drop the `XXX` prefix here. Also can you add a bit explanation of what this flag is doing under the hood at the beginning?